guides/model-state.md

```diff
@@ -0,0 +1,13 @@
+## Avoid Local/UI state on Models
+
+Models live on the `store` service, and all services hold "application state."
+UI or local state should not be entangled with application state by adding
+it to a model.
```

Comment:
> the intended sentiment is to not set UI (non-persisted) state on models in particular.

Yes, I 100% agree with this sentiment.

There's an effort in the original text) to define terminology like "UI state", "Local state", "Application state", etc... which I felt are not standardized and agreed upon in terms of definition. In particular, the term "application state" could be understood to mean a number of things, some of which would result in the various best practices being conflicting or self-contradictory.